kura/org.eclipse.kura.api/META-INF/MANIFEST.MF

```diff
@@ -29,13 +29,13 @@ Export-Package: org.eclipse.kura;version="1.3.0",
  org.eclipse.kura.gpio;version="1.0.1",
  org.eclipse.kura.linux.udev;version="1.0.0",
  org.eclipse.kura.message;version="1.1.0",
- org.eclipse.kura.net;version="1.2.1",
+ org.eclipse.kura.net;version="1.2.2",
```

Comment:
I think that since we are adding methods and deprecating something, the micro change in the APIs is not sufficient. We need at least to change the minor.